docs/testing-with-codeception.md

```diff
@@ -0,0 +1,746 @@
+
+### Advanced usage
+
+There are multiple optional parameters to customize the running command, eg:
+
+```sh
+composer dev-tools codecept [-p PATH/TO/TESTS] [-b BROWSER] run [TEST-SUITE] [[TestClass]:testMethod]
+```
+
+- `-p/--path` defines the directory where tests exists. Omit to use the `tests` root directory, or the defined test directory/directories in your configuration file.
+- `-b/--browser` defines which browser to use for acceptance tests. Omit to use the default (or configured) browser. Options are `chrome` (default), `firefox`, and `edge` (although not quite functional yet).
+- `TEST-SUITE` references the name of the test suite to run, typically one of the `*.suite.yml` files in the tests directory. Omit to run all found test suites.
+- `TestClass` references one of the test classes in the specified suite. Omit to run all tests within the suite(s).
+- `testMethod` references a single test method within the specified test class. Omit to run all test method within specified test class(es)/suite(s).
+
+Altis `codecept` command proxies commands to the `codeception` cli, except for the `-b` and `-p` cli params, so you'll be able to execute advanced commands and utilize more of what Codeception has to offer as needed, eg: generate a suite or a test:
+
+```sh
+composer dev-tools codecept generate g:cest TEST-SUITE TestClassName
+```
+
+To pass arbitary options to `codeception` command, eg: `-vv` to enable verbose mode, use a double hyphen `--` to split those as follows:
+
+```sh
+composer dev-tools codecept -- -vv
+```
+
+When you invoke the `codecept` command, this happens in the background:
+- Altis looks within the tests directory/directories for test suites files, and runs through each suite one by one in separate threads.
+- For suites using the WebDriver module, Altis boots up a docker container with a headless browser to execute those tests, based on Selenium standalone web driver containers.
+- For suites using the WPDb module, Altis sets up test databases, and seeds them with a bundled sample dump file, which can be overridden in `composer.json` if need be.
+- Tests output, eg: failed tests screenshots/html snapshots, and debugging artifacts, are saved to `PROJECT/ROOT/tests/_output` for convenience.
+- After tests have run, Altis removes the test databases, clears test caches, and the browser container.
+
+### Continuous Integration
+
+In order to run Codeception tests in Continous Integration environments, follow [Altis documentation on setting up Continous Integration on Travis](https://docs.altis-dxp.com/dev-tools/continuous-integration/), and specify your test running command(s) as per the documentation above, typically replacing `composer dev-tools phpunit` command referenced in CI setup docs with `composer dev-tools codecept`.
```

Comment:
I used:
```
In order to run Codeception tests in Continous Integration environments, follow the [documentation on setting up Continous Integration on Travis](https://docs.altis-dxp.com/dev-tools/continuous-integration/), and specify your test running command(s) as per the documentation above, typically using `composer dev-tools codecept run` instead of / in addition to `composer dev-tools phpunit` as explained in the docs.
```

Specifically stating that both can co-exist.